client/src/components/overview/productInfoAddComps/Description.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect, forwardRef } from 'react';
+import {
+  Modal, ModalBackground, DescWrapper, ShareWrap,
+} from '../overviewStyles';
+import { useAppContext } from '../../AppContext';
+import Stars from './Stars';
+import ShareView from './ShareView';
+
+const Description = forwardRef(({ currentProduct }, refRatings) => {
+  const [title, setTitle] = useState('');
+  const [category, setCategory] = useState('');
+  const [description, setDescription] = useState('');
+  const [showShare, setShowShare] = useState(false);
+  const [showStars, setShowStars] = useState(true);
+
+  // const { averageRatings, totalRatings } = useContext(AppContext);
+  const [averageRatings, setAverageRatings] = useState(2.5);
+  const [totalRatings, setB] = useState(4);
+
+  useEffect(() => {
+    if (Object.keys(currentProduct).length) {
+      setCategory(`Category: ${currentProduct.category}`);
+      setDescription(currentProduct.description);
+      setTitle(currentProduct.name);
+    }
+  }, [currentProduct]);
+
+  useEffect(() => {
+    if (showShare === false) {
+      document.body.style.overflow = 'unset';
```

Comment:
This is a minor suggestion that directly modifying DOM element should be avoided in react. Conditionally rendered styled components could be a good substitute.